identify/identify.py

```diff
@@ -257,7 +277,7 @@ def license_id(filename: str) -> Optional[str]:
             return spdx
 
         # skip the slow calculation if the lengths are very different
-        if norm and abs(len(norm) - len(norm_license)) / len(norm) > .05:
+        if norm and abs(len(norm) - len(norm_license)) / len(norm) > 0.05:
             continue
```

Comment:
I didn't make a few of these changes, they were done via precommit or black.. Maybe the precommit config changed recently?